packages/katello/katello/katello-service

```diff
@@ -3,15 +3,14 @@
 require 'optparse'
 
 ACTIONS = ['restart', 'stop', 'start', 'status', 'list', 'enable', 'disable']
-COMMAND = '/usr/sbin/service-wait'
+COMMAND = 'systemctl'
```

Comment:
I think this could be removed or the other case of `systemctl` should also use `COMMAND`